solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -96,6 +96,12 @@ solana_program::custom_panic_default!();
 
 #[anchor_lang::program]
 pub mod solana_ibc {
+    use std::time::Duration;
+
+    use ::ibc::core::channel::context::SendPacketValidationContext;
+    use ::ibc::core::connection::types::ConnectionEnd;
+    use ::ibc::core::host::types::identifiers::ConnectionId;
+    use ::ibc::core::host::ExecutionContext;
```

Comment:
Those are all re-exported in `ibc` module. I’d just use `ibc::Foo` directly where those types are used.